editor/test/resources/buildtime_conversion/game.project

```diff
@@ -1,6 +1,5 @@
 [project]
 title = Game Project Properties
-dependencies = http://test.com/test.zip
```

Comment:
Any invalid dependencies were stalling for 75 seconds each until they finally gave up and threw a `SocketTimeoutException`.